src/resource_api/schema.py

```diff
@@ -190,8 +190,21 @@ def get_schema(self):
 
 
 class IndexableField(BaseSimpleField):
+    """
+    Base class for most of the primitive fields.
+
+    choices (list|tuple = None)
+        Predefined list of possible values for the field. ValidationError will be raised if field value doesn't
+        match any of the defined values.
+    choice_labels (list|tuple = None)
```

Comment:
I like the idea with human readable labels. But make it a dict then - not two list parameters. Cause I already foresee the situation when someone adds a choice somewhere in the middle of the list and forgets to add the label exactly at the same spot of another list - and WOOOHA! a month passes and no one has a clue why the data that was coming from UI is so inappropriate.